gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -1668,7 +1668,7 @@ public static SnapshotListOption fields(SnapshotField... fields) {
   Snapshot getSnapshot(String snapshot, SnapshotOption... options);
 
   /**
-   * Lists all snapshots.
+   * Lists snapshots.
```

Comment:
Would be nice to rephrase the javadoc as "Lists both zone and regions snapshots" or something like that.